datasets/ar_sarcasm/README.md

```diff
@@ -0,0 +1,156 @@
+---
+annotations_creators:
+- no-annotation
+language_creators:
+- Ibrahim Abu Farha
+languages:
+- ar
+licenses:
+- mit
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- sentiment-classification
+- text-classification-other-sarcasm-detection
+---
+
+# Dataset Card for ArSarcasm
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+### Dataset Summary
+
+ArSarcasm is a new Arabic sarcasm detection dataset.
+The dataset was created using previously available Arabic sentiment analysis
+datasets ([SemEval 2017](https://www.aclweb.org/anthology/S17-2088.pdf)
+and [ASTD](https://www.aclweb.org/anthology/D15-1299.pdf)) and adds sarcasm and
+dialect labels to them.
+
+The dataset contains 10,547 tweets, 1,682 (16%) of which are sarcastic.
+
+For more details, please check the paper
+[From Arabic Sentiment Analysis to Sarcasm Detection: The ArSarcasm Dataset](https://www.aclweb.org/anthology/2020.osact-1.5/)
+
+### Supported Tasks and Leaderboards
+
+https://sites.google.com/view/ar-sarcasm-sentiment-detection/
+
+### Languages
+
+Arabic (multiple dialects)
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
```

Comment:
Can you add one example from the dataset here please ?

You can get one by using `load_dataset` and taking the first sample `dataset["train"][0]` for example